docker/tensorflow/mivisionx-with-tensorflow.dockerfile

```diff
@@ -55,4 +55,8 @@ RUN git clone https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX
 RUN cd ~/models/research/ && \
 protoc object_detection/protos/*.proto --python_out=. && \
 cp object_detection/packages/tf2/setup.py . && \
-pip3 install --use-feature=2020-resolver .
+python -m pip install .
+
+#upgrade pip
+RUN pip3 install --upgrade pip
+
```

Comment:
@kiritigowda can you change this to the version you used in the setup script. That should fix this issue